.pre-commit-config.yaml

```diff
@@ -34,6 +34,10 @@ repos:
         name: Run Ruff (format) on Tools/build/check_warnings.py
         args: [--check, --config=Tools/build/.ruff.toml]
         files: ^Tools/build/check_warnings.py
+      - id: ruff-check
+        name: Run Ruff (lint) on Tools/wasm/
+        args: [--exit-non-zero-on-fix, --config=Tools/wasm/.ruff.toml]
+        files: ^Tools/wasm/
```

Comment:
Let's move this up, so we keep all the `ruff-check`s together in the first half, and `ruff-format`s in the second (compare `Doc/`).